### PR TITLE
Add docs about i18n

### LINKED ISSUE #9: Update docs
- [ ] - i18n
- [ ] - techMap
- [x] - move out imports syntax to https://github.com/bem-sdk/bem-import-notation

### PR DESCRIPTION
**Pull request by Yeti-or**
@awinogradov check this docs
fix #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,75 @@ import BlockElem from 'b:block e:elem m:modName=modVal1|modVal2';
 ## Options
 
 - __naming__: [bem-naming](https://en.bem.info/toolbox/sdk/bem-naming) overrides
-- __levels__<Array>: paths to components declarations
-- __techs__<Array>: list of techs extensions for require in runtime
+- __levels__ <Array>: paths to components declarations
+- __techs__ <Array>: list of techs extensions for require in runtime, `['js']` by default
+- __techMap__ <Object>: mapping of techs to extensions. Example: `{ 'js' : ['react.js', 'react.ts', 'react.es'], 'css' : ['post.css'] }`
```
Comment (awinogradov) [inline README.md:42]: react everywhere?

File: README.md [hunk 2]
```diff
@@ -41,7 +41,75 @@ import BlockElem from 'b:block e:elem m:modName=modVal1|modVal2';
...
+- __langs__ <Array>: list of langs in which resloves '.i18n' tech
```
Comment (awinogradov) [inline README.md:43]: Mistake?

File: README.md [hunk 3]
```diff
@@ -41,7 +41,75 @@ import BlockElem from 'b:block e:elem m:modName=modVal1|modVal2';
...
+
+## i18n
+
+`.i18n` - represent special technology that provides the opportunity to localize components.
+
+For correct working you need to install `bem-i18n` inside your project.
```
Comment (awinogradov) [inline README.md:49]: It can't be by default?

File: README.md [hunk 4]
```diff
@@ -41,7 +41,75 @@ import BlockElem from 'b:block e:elem m:modName=modVal1|modVal2';
...
+
+```
+npm i -S bem-i18n
+```
+
+On file system:
+
+```
+blocks/attach/
```
Comment (awinogradov) [inline README.md]: You need to use modern naming here

File: README.md [hunk 5]
```diff
@@ -41,7 +41,75 @@ import BlockElem from 'b:block e:elem m:modName=modVal1|modVal2';
...
+$ cat common.blocks/attach/attach.i18n/tr.js
+module.exports = {
+    "attach": {
+        "button-text": "Dosya seç",
+        "no-file": "dosya seçilmedi"
+    }
+};
+```
+
+inside `attach.react.js`:
```
Comment (awinogradov) [inline README.md]: And everywhere

File: README.md [hunk 6]
```diff
@@ -41,7 +41,75 @@ import BlockElem from 'b:block e:elem m:modName=modVal1|modVal2';
...
+
+```js
+import i18n from `b:attach t:i18n`
+
+console.log(i18n('button-text')) // → Dosya seç
+```
+
+`babel-import` transpiles such code to
```
Comment (awinogradov) [inline README.md:87]: I think we don't need to explain this here

**Review by awinogradov — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (Yeti-or) [reply README.md:43]: what do you mean?

File: README.md (hunk 1 above)
Comment (Yeti-or) [reply README.md:42]: yeap

File: README.md (hunk 3 above)
Comment (Yeti-or) [reply README.md:49]: by default ? right know you need to do this

File: README.md (hunk 4 above)
Comment (Yeti-or) [reply README.md]: you mean `blocks/Attach` ?

File: README.md (hunk 6 above)
Comment (Yeti-or) [reply README.md:87]: so then where we need to explain it?

**Comment by Yeti-or** [comment]
I fixed naming

**Pull request merged**